cmd/sanssh/main.go

```diff
@@ -62,14 +62,18 @@ const (
 )
 
 var (
-	defaultTimeout = 30 * time.Second
+	defaultDialTimeout = 10 * time.Second
+	defaultIdleTimeout = 15 * time.Minute
 
 	proxyAddr = flag.String("proxy", "", fmt.Sprintf(
 		`Address (host[:port]) to contact for proxy to sansshell-server.
 %s in the environment can also be set instead of setting this flag. The flag will take precedence.
 If blank a direct connection to the first entry in --targets will be made.
 If port is blank the default of %d will be used`, proxyEnv, defaultProxyPort))
-	timeout          = flag.Duration("timeout", defaultTimeout, "How long to wait for the command to complete")
+	// Deprecated: --timeout flag is deprecated. Use --idle-timeout or --dial-timeout instead
+	_                = flag.Duration("timeout", defaultDialTimeout, "DEPRECATED. Please use --idle-timeout or --dial-timeout instead")
```

Comment:
Right now this is silently ignored.

I think we should remove it entirely. This main function is an example of how users of sansshell should implement sanssh, so we can get away with breaking changes.

The other option is sending a message to stderr when this is set and possibly exiting with an error code.

Reply:
Updated to run log.Fatalf when this is set